src/Altinn.App.Api/Controllers/InstancesController.cs

```diff
@@ -475,66 +462,100 @@ public class InstancesController : ControllerBase
             return Created(url, instance);
         }
 
-        private async Task CopyDataFromSourceInstance(ApplicationMetadata application, Instance targetInstance, Instance sourceInstance)
+        /// <summary>
+        /// This method handles the copy endpoint for when a user wants to create a copy of an existing instance.
+        /// The endpoint will primarily be accessed directly by a user clicking the copy button for an archived instance.
+        /// </summary>
+        /// <param name="org">Unique identifier of the organisation responsible for the app</param>
+        /// <param name="app">Application identifier which is unique within an organisation</param>
+        /// <param name="instanceOwnerPartyId">Unique id of the party that is the owner of the instance</param>
+        /// <param name="instanceGuid">Unique id to identify the instance</param>
+        /// <returns>A <see cref="Task{TResult}"/> representing the result of the asynchronous operation.</returns>
+        /// <remarks>
+        /// The endpoint will return a redirect to the new instance if the copy operation was successful.
+        /// </remarks>
+        [Authorize]
+        [HttpGet("{instanceOwnerPartyId:int}/{instanceGuid:guid}/copy")]
```

Comment:
Should prefix this path with legacy to make it obvious that this should not be used.
Exclude this from OpenAPI specs and mark the method as obsolete.

Reply:
New endpoint would be: /legacy/instances/<org>/<app>/<ownerPartyID>/<instanceGUID>/copy

Reply:
Made it: `/<org>/<app>/legacy/instances/<ownerPartyID>/<instanceGUID>/copy`